system-int-test/test/cmr/system_int_test/ingest/subscription_ingest_test.clj

```diff
@@ -758,6 +763,7 @@
                      {:SubscriberId "post-user"
                       :Name "a different subscription with native-id"
                       :native-id "another-native-id"
+                      :Query "polygon=1,2,3"
```

Comment:
nit: throughout the PR, `polygon=1,2,3` is not a valid polygon value. Change it to something valid like `platform=x`